hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -268,6 +268,7 @@ public static boolean isReadOnly(
     case SnapshotDiff:
     case ListSnapshotDiffJobs:
     case TransferLeadership:
+    case CustomUnknownCommand:
```

Comment:
IMO we do not need implement new request handler. This is crashing since IllegalStateException is not caught.
We can use IOException or OMException (sub-class of IOException)... which will be handled at caller and will return error response automatically.

Reply:
Thank you @sumitagrawl for the review, noted your point - will be making the suggested changes.

I was mostly looking from the perspective that OM should be creating a response indicating that it does not know how to process the incoming unknown request.